identity-service/src/routes/userEvents.js

```diff
@@ -21,4 +22,34 @@ module.exports = function (app) {
       return successResponse({})
     }
   }))
+
+  /* Updates that UserEvent table w/ if the user has sign in w/ native mobile
+  *
+  * @param {boolean} hasSignedInNativeMobile   If the user has signed in w/ native mobile
+  */
+  app.post('/userEvents', authMiddleware, handleResponse(async (req, res) => {
```

Comment:
should this route let us flip the hasSignInNativeMobile value to false? (probs fine)

Reply:
Yeah, I thought about it and I didn't see a case where we would ever flip it to false.